corehq/apps/auditcare/tests/test_auditcare_migration.py

```diff
@@ -121,19 +123,22 @@ def setUpClass(cls):
             f'{datetime(2021,5,15)}_{datetime(2021,5,16)}',
             f'{datetime(2021,5,1)}_{datetime(2021,5,2)}',
         ]
-        AuditcareMigrationMeta(key=cls.errored_keys[0], state=AuditcareMigrationMeta.ERRORED).save()
-        AuditcareMigrationMeta(key=cls.errored_keys[1], state=AuditcareMigrationMeta.ERRORED).save()
-
         return super().setUpClass()
 
     @classmethod
     def tearDownClass(cls):
         cache.delete(AuditCareMigrationUtil().start_key)
+        delete_couch_docs(cls.couch_doc_ids)
+        return super().tearDownClass()
+
+    def tearDown(self):
         NavigationEventAudit.objects.all().delete()
         AccessAudit.objects.all().delete()
         AuditcareMigrationMeta.objects.all().delete()
```

Comment:
These will automatically be removed since `TestCase` runs each test in a transaction which is rolled back after the test is run. I think the `tearDown` method can be removed.

I didn't look closely to see if there were any other tests that are doing unnecessary SQL in `tearDown`, but would be worth removing them if they exist since it makes the test take longer.